api/base/generic_bulk_views.py

```diff
@@ -87,6 +88,11 @@ def get_requested_resources(self, request, request_data):
         if len(resource_object_list) != len(request_data):
             raise ValidationError({'non_field_errors': 'Could not find all objects to delete.'})
 
+        if column_name == 'guids___id':
```

Comment:
Since you've made a change to our one of our base views, there should definitely be tests related to the change.